### ISSUE TITLE
AQUARIUM (2515070)

### ISSUE THREAD

**Issue by waspennator** [ValveSoftware/Proton#7704]
# Compatibility Report
- Name of the game with compatibility issues: AQUARIUM
- Steam AppID of the game: 2515070

## System Information
- 1 TB OLED Steam deck LE 
- Proton version: 9.0-1 and Experimental 

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game has 2 major issues.

1: I'm pretty sure something is supposed to be playing here looking at the gameplay footage on yt, but on my deck at least, I just get a black void with silence, though this is merely a visual issue as the main menu still works

![download](https://github.com/ValveSoftware/Proton/assets/78768499/d7c595b9-5b5d-427c-aed4-c7fa8fe1a563)

2:  The game works up until you get to the first fmv (A bit after you are introduced to the new maid Aqua), on Proton Experimental and 9.0-1, the fmv seems to stutter a bit before fast forwarding a bit to the correct spot, but it also outright freezes if you let it finish playing or try to skip it. If you drop down to 8.0-5, it plays and skips just fine.

[steam-2515070.log](https://github.com/ValveSoftware/Proton/files/15211184/steam-2515070.log)

## Reproduction
Start the game and look at the title screen, then play through the main game and play through the normal story until you get to the fmv in question

**Comment by waspennator** [comment]
Still occurring on latest Experimental, checking through the logs, I see some stuff about gstreamer failing to load plug-ins, but I'm not sure if that relates to issue number 2 or not. Curious if this has something to do with it though.

"(wine:42999): GStreamer-Video-CRITICAL **: 16:47:11.806: gst_video_info_from_caps: assertion 'gst_caps_is_fixed (caps)' failed"

"(wine:42999): GStreamer-Base-WARNING **: 16:47:12.079: gldownloadelement0: transform_caps returned caps which are not a real subset of the filter caps
0:00:01.256368403 42999 0xdec0f9c0 ERROR          basetransform gstbasetransform.c:482:gst_base_transform_transform_caps:<glcolorconvertelement1> transform_caps returned caps video/x-raw(memory:GLMemory), format=(string)RGBA, width=(int)320, height=(int)240, multiview-mode=(string)mono, multiview-flags=(GstVideoMultiviewFlagsSet)0:ffffffff:/right-view-first/left-flipped/left-flopped/right-flipped/right-flopped/half-aspect/mixed-mono, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)30/1, texture-target=(string)2D, interlace-mode=(string)progressive; video/x-raw(memory:GLMemory), format=(string)RGBA, width=(int)320, height=(int)240, framerate=(fraction)30/1, texture-target=(string)2D, multiview-mode=(string)mono, multiview-flags=(GstVideoMultiviewFlagsSet)0:ffffffff:/right-view-first/left-flipped/left-flopped/right-flipped/right-flopped/half-aspect/mixed-mono, pixel-aspect-ratio=(fraction){ 1/1, [ 1/2147483647, 2147483647/1 ] }, interlace-mode=(string)progressive which are not a real subset of the filter caps video/x-raw(memory:GLMemory), format=(string)RGBA, width=(int)320, height=(int)240, multiview-mode=(string)mono, multiview-flags=(GstVideoMultiviewFlagsSet)0:ffffffff:/right-view-first/left-flipped/left-flopped/right-flipped/right-flopped/half-aspect/mixed-mono, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)30/1, texture-target=(string)2D; video/x-raw(memory:GLMemory), format=(string)RGBA, width=(int)[ 1, 2147483647 ], height=(int)[ 1, 2147483647 ], framerate=(fraction)30/1, texture-target=(string)2D, multiview-mode=(string)mono, multiview-flags=(GstVideoMultiviewFlagsSet)0:ffffffff:/right-view-first/left-flipped/left-flopped/right-flipped/right-flopped/half-aspect/mixed-mono, pixel-aspect-ratio=(fraction)[ 1/2147483647, 2147483647/1 ]"

Copied a part of it cause it's pretty long, but it should be easy to ctrl+f to the specific section to check, might run a log with 8.0-5 to see any differences.

**Comment by waspennator** [comment]
Update, testing with 8.0-5, missing plug-ins are unrelated to the regression, checking the log it tosses up stuff about getinputtype, setoutputtype, getoutputtype, etc, but I also see this stuff in the logs.

(wine:146199): GStreamer-Base-WARNING **: 21:53:27.789: gldownloadelement0: transform_caps returned caps which are not a real subset of the filter caps

[steam-2515070.log.txt](https://github.com/user-attachments/files/15746389/steam-2515070.log.txt)

**Comment by waspennator** [comment]
Latest proton Experimental bleeding edge version seems to have broken the fmv even more, now it doesn't even render at all.

[steam-2515070.log.txt](https://github.com/user-attachments/files/16114847/steam-2515070.log.txt)

**Comment by AJuujarvi** [comment]
> Latest proton Experimental bleeding edge version seems to have broken the fmv even more, now it doesn't even render at all.
> 
> [steam-2515070.log.txt](https://github.com/user-attachments/files/16114847/steam-2515070.log.txt)

I narrowed it down to a DXVK regression and reported it here: https://github.com/doitsujin/dxvk/issues/4122

**Comment by waspennator** [comment]
Seems like the regression has been fixed up on dxvks end, dunno about the other issues that the fmv had reported earlier in the thread though.

**Comment by waspennator** [comment]
Behavior is now back to how it was before, fmv stutters a bit before catching up, game softlocks at the end of the fmv

**Comment by waspennator** [comment]
The lag spike that occurs during the beginning of the fmv seems to be gone now, but it still softlocks once it ends (manually skipping or automatically), compared to 8.0-5

The new media source update hasn't fixed the main menu fmv not playing or the fmv softlock.

**Comment by waspennator** [comment]
The softlock that happens when the fmv ends seems to finally be fixed now, so that just leaves the main menu fmv not playing. Then all the issues I've seen with this game will be resolved.

At the very least the game can be bumped out of unsupported (maybe with a notice about minor visual issues), since the main menu fmv isn't gamebreaking.

**Comment by waspennator** [comment]
The fmv on the main menu is now playable as of proton Experimental bleeding edge, though a major regression has been introduced, the game now silently crashes if you try to start a new game.


[steam-2515070.log.txt](https://github.com/user-attachments/files/19968822/steam-2515070.log.txt)

https://gitlab.winehq.org/wine/wine/-/merge_requests/7676

**Comment by alasky17** [comment]
@waspennator Thank you for the report! Unfortunately, this is part of the "progress" of getting the videos to actually play instead of being skipped entirely 😅  - we are working on a real fix, and in the mean time folks should be able to continue to use 9.0-4.  Thanks again for testing and the speedy report!

**Comment by waspennator** [comment]
Still occurring with proton Experimental, however I'm noticing the fmvs in #8973 aren't playing, dunno if it's borked differently compared to this or if the fmv needs to be reencoded, both games made by the same dev

**Comment by waspennator** [comment]
@alasky17 

As of the latest proton Experimental update, the game finally plays the title screen fmv and gets ingame without crashing!

However im sorry to say but this needs to be temporarily removed from the now playable part of the changelog as the game gets stuck after the fmv that plays after you are introduced to maid aqua on a black screen. If that was fixed I could finally close this issue.

[steam-2515070.log](https://github.com/user-attachments/files/23136330/steam-2515070.log)

**Comment by AJuujarvi** [comment]
@waspennator Hello and thank you for the report! Looking into it, I believe I found what seems to be causing the issue and will be looking into it. In the meantime, I found that an Nvidia Desktop, and a Deck in Desktop Mode will both play the game correctly, getting past the video you described and playing the "Chapter 1" intro screens before going into a conversation between Aqua and Marine.

**Comment by waspennator** [comment]
Game is so close to being perfect but everytime the issue im running into gets fixed, it's replaced with a different one. The freeze after the fmv that plays before ch1 is finally gone...... but now the damn title screen fmv is doing the test bars like Okayunyumu.

[steam-2515070.log](https://github.com/user-attachments/files/28133477/steam-2515070.log)

**Comment by alasky17** [comment]
@waspennator I believe it is showing the test pattern because we fixed the video and it was just skipped before.  Test pattern should eventually be replaced by a transcoded video, but sadly I have no way to manually trigger that.

**Comment by waspennator** [comment]
> [@waspennator](https://github.com/waspennator) I believe it is showing the test pattern because we fixed the video and it was just skipped before. Test pattern should eventually be replaced by a transcoded video, but sadly I have no way to manually trigger that.

No, I remembered after you fixed the fmv not playing, it started playing just fine for all my later tests, it just started doing the test bars with my recent tests. Also because Okayunyumus fmv is playing perfectly fine despite being newer.
